src/annotation/InvalidData.java

```diff
@@ -0,0 +1,28 @@
+package annotation;
+
+import java.lang.reflect.Field;
+
+public class InvalidData {
+
+    private final Class<?> aClass;
```

Comment:
Vahan jan, if you need the only display the name of the class and name of the field, you can have String className and String fieldName instead of Class<?> aClass and Field field